scripts/circleci/release/installScalyrAgentV2.sh

```diff
@@ -147,6 +150,11 @@ VERSION=
 # If non-empty, the scalyr server to use for uploading logs.
 SCALYR_SERVER=
 
+REPOSITORY_URL="https://scalyr-repo.s3.amazonaws.com/{ % REPLACE_REPOSITORY_TYPE % }"
```

Comment:
I would add a comment that this should be replaced by the `create-agent-installer.sh` script.

Reply:
Done.

Reply:
On that note, to be on the safe side, we should probably add additional check to AMI tests which verifies installer file doesn't container this place holder string when AMI tests are operating with installer script file or URL and not directly with packages.